### PR TITLE
resolving mhkit-matlab live example html issue

### PR DESCRIPTION
**Pull request by kmruehl**
The MHKiT-MATLAB Live Example `*.mlx` files have to be saved as `*.html` files, and copied into the `mhkit\docs\mhkit-matlab` documentation directory. This means that the MHKiT-MATLAB submodule _cannot_ be used to generate `*.html` files for the Live Examples, but the submodule _can_ be used for API documentation.

### CODE REVIEW

**Comment by kmruehl** [comment]
@rpauly18 this means you can remove the `*.html` files from the MHKiT-MATLAB repository, [in this directory](https://github.com/MHKiT-Software/MHKiT-MATLAB/tree/master/examples).

**Comment by kmruehl** [comment]
@SeanPluemer the last commit added terminology to partially resolve #19. You'll need to add a variable column to fully resolve the issue. I also made some minor edits to your `citeus.rst` to resolve some Sphinx warnings and add the the DOE to the Bibtex citation.

**Comment by rpauly18** [comment]
@kmruehl just so I understand, this change does not fix the issue of the index disappearing for the MATLAB examples, correct?

**Comment by kmruehl** [comment]
> 
> 
> @kmruehl just so I understand, this change does not fix the issue of the index disappearing for the MATLAB examples, correct?

Correct. I do not know how to resolve the index issue for MATLAB Live files. This is just moving the `*.mlx` and `*.html` files into the docs directory instead of using the submodule.

**Comment by ssolson** [comment]
Kelly could you pull the MHKiT-Python submodule to incorporate the link changes you just approved then push those htmls up for this PR?

File: docs/_sources/citeus.rst.txt [hunk 1]
```diff
@@ -14,13 +14,13 @@ MHKiT-Python
...
+        doi       = {10.11578/dc.20200113.1}
     }
 
-.. _pubs:
 
 MHKiT-MATLAB
---------
+----------------
 | MHKiT-Python DOI: `10.11578/dc.20200110.1 <https://www.osti.gov/doecode/biblio/33549>`_
 | If you desire to cite `MHKiT-MATLAB` in a publication, please use the following bibtex key:
```
Comment (SeanPluemer) [inline docs/_sources/citeus.rst.txt:25]: Should this be BibTeX?

File: docs/_sources/citeus.rst.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Cite Us
 ============
 MHKiT-Python
---------
+----------------
 | MHKiT-Python DOI: `10.11578/dc.20200113.1 <https://www.osti.gov/doecode/biblio/33583>`_
 | If you desire to cite `MHKiT-Python` in a publication, please use the following bibtex key:
```
Comment (SeanPluemer) [inline docs/_sources/citeus.rst.txt:6]: Should this be BibTeX?

File: docs/source/citeus.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Cite Us
 ============
 MHKiT-Python
---------
+----------------
 | MHKiT-Python DOI: `10.11578/dc.20200113.1 <https://www.osti.gov/doecode/biblio/33583>`_
 | If you desire to cite `MHKiT-Python` in a publication, please use the following bibtex key:
```
Comment (SeanPluemer) [inline docs/source/citeus.rst:6]: Should this be BibTeX?

File: docs/source/citeus.rst [hunk 4]
```diff
@@ -14,13 +14,13 @@ MHKiT-Python
...
+        doi       = {10.11578/dc.20200113.1}
     }
 
-.. _pubs:
 
 MHKiT-MATLAB
---------
+----------------
 | MHKiT-Python DOI: `10.11578/dc.20200110.1 <https://www.osti.gov/doecode/biblio/33549>`_
 | If you desire to cite `MHKiT-MATLAB` in a publication, please use the following bibtex key:
```
Comment (SeanPluemer) [inline docs/source/citeus.rst:25]: Should this be BibTeX?

File: docs/_sources/citeus.rst.txt (hunk 1 above)
Comment (ssolson) [reply docs/_sources/citeus.rst.txt:25]: Looks to me that bibtex should be BibTeX.

**Comment by kmruehl** [comment]
@ssolson pulled in the updated submodule to resolve example data links
@SeanPluemer updated to BibTeX, and minor formatting change

**Pull request merged**